build.gradle

```
@@ -21,6 +21,11 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+	implementation group: 'org.springdoc', name: 'springdoc-openapi-starter-webmvc-ui', version: '2.0.3'
```

We should be consistent and have the same import format, using Gradle short version of th import.
```suggestion
implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.0.3'
```